app/javascript/components/patient/assessment/steps/SymptomsAssessment.js

```diff
@@ -265,12 +266,12 @@ class SymptomsAssessment extends React.Component {
         <Card.Body>
           {this.props.current_user && (
             // Only allow users, not monitorees, to edit the report date
-            <Form.Group>
+            <Form.Group controlId="reported_at">
               <Form.Row>
                 <Form.Label className="input-label mr-2 pt-1">Symptom Report for Date:</Form.Label>
                 <DateInput
                   id="reported_at"
-                  date={this.props.assessment.reported_at ?? moment().toDate()}
+                  date={this.state.reportState.reported_at}
```

Comment:
In testing, when I click the "Add new Report" button, the Symptom Assessments panel glitches out with the error: `Uncaught RangeError: Invalid time value`. It appears to trace back to line 272. Can anyone else reporduce this?

Reply:
hmm I tested this again and I'm not seeing it

Reply:
I am also not seeing this

Reply:
confirmed this occurs on firefox NOT chrome.  ended up being that the date format we use just for reports isn't supported in firefox.  fixed this here: https://github.com/SaraAlert/SaraAlert/pull/1248/commits/a08e5f97f9ca9ce84d6c884b935730650563fcdc